src/utils/exportUtils/Exporter.ts

```diff
@@ -75,9 +75,15 @@ export default abstract class Exporter {
         this.files.push(file);
     }
 
+    public santizeFileName(filename: string): string {
+        filename = filename.replace(/[^a-z0-9áéíóúñü.,_-]/gim, "");
+        filename = filename.replace(/[ ]/gim, "-");
+        return filename.trim();
+    }
```

Comment:
JSONExporter and PlainTextExporter both extend this class so they don't need their own copies of this method and will instead inherit it. Though the method should probably be `protected`

Reply:
Now done ..